dysnix/geth/templates/s3-secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if or .Values.initFromS3.enabled .Values.syncToS3.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "geth.fullname" . }}-s3-secret
+data:
```

Comment:
Just left it in `data` cuz changes to secrets will be visible in `helm diff`

Reply:
Hm, `helm diff` doesn't display secrets' values, as long as `  --show-secrets` option is missed.

Reply:
Using `data` instead of `stringData` I can capture this in helm diff output.
```
geth, geth-2-s3-secret, Secret (v1) has changed:
  apiVersion: v1
  kind: Secret
  metadata:
    name: geth-2-s3-secret
  data:
-   AWS_ACCESS_KEY_ID: '-------- # (61 bytes)'
+   AWS_ACCESS_KEY_ID: '++++++++ # (62 bytes)'
    AWS_SECRET_ACCESS_KEY: 'REDACTED # (40 bytes)'
    S3_ENDPOINT_URL: 'REDACTED # (30 bytes)'
```
I just need to see whether it is changed or not. Don't need secret values to be present in output.

When using `stringData` helm diff does not provide info whether secret has changed or not.

Reply:
I cannot reproduce this using helmfile and dysnix/raw chart with following values
```
templates:
  - |
    apiVersion: v1
    stringData:
      foo: newbar
    kind: Secret
    metadata:
      name: {{ .Release.Name }}
    type: Opaque
```
I change `foo`'s value and see 
```
-   foo: '-------- # (3 bytes)'
+   foo: '++++++++ # (6 bytes)'
```

Reply:
Seems, [related issue](https://github.com/databus23/helm-diff/issues/302) has been fixed ~ a year ago.